pkg/istiod/common.go

```diff
@@ -214,6 +217,22 @@ func NewIstiod(kconfig *rest.Config, kclient *kubernetes.Clientset, confDir stri
 	// Actual files are loaded by galley/pkg/src/fs, which recursively loads .yaml and .yml files
 	// The files are suing YAMLToJSON, but interpret Kind, APIVersion
 
+	// This is the 'mesh' file served by Galley - not clear who is using it, ideally we should drop it.
```

Comment:
Can you raise and issue and reference it with a TODO here?

Reply:
Added it to the 'action items' doc - if anyone wants to work on it they can create an issue as well, for now we have a workaround.